tcms/testruns/static/testruns/js/get.js

```diff
@@ -447,6 +447,35 @@ function getExpandArea(testExecution) {
             historyContainer.append(renderHistoryEntry(h))
         })
     })
+
+    container.find('.change-status-button').click(function () {
+        const statusId = $(this).attr('data-status-id')
+
+        const comment = editor.value().trim()
```

Comment:
`ReferenceError: editor is not defined` - again this hasn't been tested manually and fails so I can't merge.

Reply:
I changed the place of the handler and should be ok now.